docs/alternative_runtimes.md

```diff
@@ -15,19 +12,15 @@ $ minikube start \
 To use [CRI-O](https://github.com/kubernetes-incubator/cri-o) as the container runtime, run:
 
 ```shell
-$ minikube start \
-    --network-plugin=cni \
-    --enable-default-cni \
-    --container-runtime=cri-o
+$ minikube start --container-runtime=cri-o
```

Comment:
Do you mind updating the integration test to also omit these flags? This way this change gets tested properly.

See test/integration/util/util.go line ~211 & 213.

Reply:
Sounds like a good idea!

Reply:
currently are we using cri-o as only container- runtime for minikube start?